backend/src/main/java/gov/cdc/usds/simplereport/api/model/ApiFacility.java

```diff
@@ -16,33 +17,33 @@ public String getName() {
 
 	public String getCliaNumber() {
 		return org.getCliaNumber();
-  }
+    }
   
-  public String getStreet() {
-		return "2797 N Cerrada de Beto";
+    public String getStreet() {
+		return org.getAddress().getStreetOne();
```

Comment:
Shouldn't we be pulling the address info from the facility itself?

Reply:
We are, it's just still named `org` because I wanted to keep the diff small and knew I was going to be redoing a bunch of this class later in the day anyway. 😸